### PR TITLE
Issue #351: Update pager components enabling class attributes on links.

### LINKED ISSUE #351: Update pager components allowing class attributes

### PR DESCRIPTION
**Pull request by aritomelo**
Fix #351

### CODE REVIEW

File: atomium/includes/common.inc [hunk 1]
```diff
@@ -646,8 +646,6 @@ function atomium_get_base_hook($hook) {
  * Discover templates based on directory name.
  *
  * It is important that directories have the name of the hook for the moment.
- *
- * @deprecated Use atomium_discover_templates()
  */
 function atomium_find_templates() {
```
Comment (drupol) [inline atomium/includes/common.inc]: If `atomium_find_templates()` is not used elsewhere, please remove it completely then.

File: atomium/includes/classes/Attributes.php [hunk 2]
```diff
@@ -322,8 +322,6 @@ public function remove($key, $value = FALSE) {
    * @param string|array $name
    *   The name of the attribute key to delete.
    *
-   * @deprecated
```
Comment (drupol) [inline atomium/includes/classes/Attributes.php:325]: If the method is not used elsewhere, please remove it.

**Review by drupol — CHANGES_REQUESTED** [review]
:crying_cat_face:

File: docker-compose.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ services:
     image: fpfis/httpd-php-dev:7.1
     working_dir: /var/www/html
     ports:
-    - 8080:8080
+    - 8080
```
Comment (drupol) [inline docker-compose.yml]: Why this ?

Comment (aritomelo) [reply docker-compose.yml]: Mistake, only needed on my local!

**Pull request merged**